packages/dom/src/dom.js

```diff
@@ -264,7 +264,8 @@ export function placeCaretAtHorizontalEdge( container, isReverse ) {
 	}
 
 	if ( includes( [ 'INPUT', 'TEXTAREA' ], container.tagName ) ) {
-		container.focus();
+		container.focus( { preventScroll: true } );
```

Comment:
This would make it so that all other calls to `placeCaretAtHorizontalEdge` also prevent scroll. 

An option could be to make the function accept another parameter. 

`placeCaretAtHorizontalEdge( container, isReverse, preventScroll )` might be too specific and its heading towards this function potentiall having loads of params. Using an optional options object might be the best tradeoff `placeCaretAtHorizontalEdge( container, isReverse, { preventScroll: true } );`.

Seems like a legit option to me! Would be good to have someone who's more familiar with the DOM package also 👍 this idea.